Casks/p/paintcode.rb

```diff
@@ -14,4 +14,11 @@
   end
 
   app "PaintCode.app"
+
+  zap trash: [
+    "~/Library/Application Support/com.pixelcut.paintcode#{version.major}",
+    "~/Library/HTTPStorages/com.pixelcut.paintcode#{version.major}",
+    "~/Library/Preferences/com.pixelcut.paintcode#{version.major}.plist",
+    "~/Library/Saved Application State/com.pixelcut.paintcode3.savedState",
```

Comment:
```suggestion
    "~/Library/Saved Application State/com.pixelcut.paintcode#{version.major}.savedState",
```

In saying that, I'm usually more in favour of use `*` in place of `version.major` in the `zap` stanza.

Reply:
Good point, thanks.